pkgs/servers/monitoring/prometheus/promscale/default.nix

```diff
@@ -29,6 +29,10 @@ buildGoModule rec {
     fi
   '';
 
+  patches = [
+    ./0001-remove-jaeger-test-dep.patch
+  ];
+
```

Comment:
Suggestion:
patches reside directly under src.

Reply:
I'm sorry, I don't understand this suggestion.

Reply:
Ah, I think that I figure this out.

Reply:
```
--- a/pkgs/servers/monitoring/prometheus/promscale/default.nix
+++ b/pkgs/servers/monitoring/prometheus/promscale/default.nix
@@ -14,6 +14,10 @@ buildGoModule rec {
     sha256 = "sha256-OMDl8RGFOMW+KNX2tNHusJY/6gLZxuWCI3c0E/oqrfE=";
   };

+  patches = [
+    ./0001-remove-jaeger-test-dep.patch
+  ];
+
   vendorSha256 = "sha256-IwHngKiQ+TangEj5PcdiGoLxQJrt/Y3EtbSYZYmfUOE=";

   ldflags = [ "-s" "-w" "-X github.com/timescale/promscale/pkg/version.Version=${version}" "-X github.com/timescale/promscale/pkg/version.CommitHash=${src.rev}" ];
@@ -29,10 +33,6 @@ buildGoModule rec {
     fi
   '';

-  patches = [
-    ./0001-remove-jaeger-test-dep.patch
-  ];
-
   meta = with lib; {
     description = "An open-source analytical platform for Prometheus metrics";
     homepage = "https://github.com/timescale/promscale";
```